### ISSUE TITLE
css loaders

### ISSUE THREAD

**Issue by Maxlufs** [roman01la/webpack-closure-compiler#11]
Hi, I have a case where in main.js, css is imported. So during the build, the setup looks like this 

``` js
module: {
  loaders: [{
    test: /\.css$/,
    include: [POSTCSS_SRC_DIR],
    loader: ExtractTextPlugin.extract('style', 'css?root=..!postcss')
  },{
    test: /\.js$/,
    include: [JS_SRC_DIR],
    loader: 'babel-loader'
  }]
},
plugins: [
      new ClosureCompilerPlugin({
      compiler: {
        language_in: 'ECMASCRIPT6',
        language_out: 'ECMASCRIPT5_STRICT',
        compilation_level: 'ADVANCED',
      },
      concurrency: 3,
    })
]
```

And it gives me error: 

``` sh
ERROR in ./src/css/main.css
Module build failed: TypeError: text.forEach is not a function
```

seems like the plugin doesn't take any non js files extracted by ExtractTextPlugin. Is there any way to ignore css files or work around this issue? 

Thank you!

**Comment by DekusDenial** [comment]
having the same issue here~~

**Comment by roman01la** [comment]
Fixed and published as v2.0.2. I've added `test` option where you can specify a pattern for file names which should be processed, default is `/\.js($|\?)/i`.

``` js
        new ClosureCompilerPlugin({
          test: /\.js($|\?)/i,
          compiler: {
            language_in: 'ECMASCRIPT6',
            language_out: 'ECMASCRIPT5',
            compilation_level: 'ADVANCED'
          },
          concurrency: 3,
        })
```
